### ISSUE TITLE
PopupView seems to follow the component with the biggest width in the view?

### ISSUE THREAD

**Issue by Nathan1258** [Mijick/PopupView#69]
Popup seems to follow the component with the biggest width in the view?

<img width="402" alt="Screenshot 2024-01-23 at 23 25 59" src="https://github.com/Mijick/PopupView/assets/53881694/ae551d63-b1a7-4b03-a2ac-66ad8abab1d6">

it works as expected when content overflows to the next line

<img width="402" alt="Screenshot 2024-01-23 at 23 27 16" src="https://github.com/Mijick/PopupView/assets/53881694/b9dd8882-015f-464e-8d1d-10f1d5c19e26">

Code:

`
      import MijickPopupView
      import SwiftUI

      struct BottomCustomPopup: BottomPopup {
          func createContent() -> some View {
              HStack(spacing: 0) {
                  Text("Witaj okrutny świecie")
                  Spacer()
                  Button(action: dismiss) { Text("Dismiss") }
              }
              .padding(.vertical, 20)
              .padding(.leading, 24)
              .padding(.trailing, 16)
          }
          func configurePopup(popup: BottomPopupConfig) -> BottomPopupConfig {
              popup
                  .horizontalPadding(20)
                  .bottomPadding(42)
                  .cornerRadius(16)
                  .tapOutsideToDismiss(true)
                  .dragGestureEnabled(true)
                  .contentIgnoresSafeArea(true)
          }
      }
`

**Comment by FulcrumOne** [comment]
Hey, @Nathan1258!

It seems to me that you put modifier `implementPopupView` in the wrong place. Could you please show me where you inserted it? 

Thanks!

**Comment by Nathan1258** [comment]
Hi, @FulcrumOne 

I've put it at the `@main` call:

```

import SwiftUI
import MijickPopupView


@main
struct OverloadApp: App {
   
    var body: some Scene {
        WindowGroup {
            Splash().implementPopupView()
        }
    }
}
```

**Comment by FulcrumOne** [comment]
Yeah, I have just checked and you are right. However, the bug is expected to be fixed in the `patch-2.2.3` branch
Could you confirm that everything works as expected there?

**Comment by Nathan1258** [comment]
> Yeah, I have just checked and you are right. However, the bug is expected to be fixed in the `patch-2.2.3` branch
> 
> Could you confirm that everything works as expected there?

Hi. Sorry for the delayed response. Yes I can confirm this branch has a fix for the issue. Thank you!
